src/components/fields/radio-button/radio-button.tsx

```diff
@@ -97,8 +104,9 @@ export const RadioButtonGroup = (props: IGenericFieldProps<IRadioButtonGroupSche
 
 	const renderToggles = () => {
 		return (
-			options.length > 0 && (
-				<FlexWrapper className={className}>
+			options.length > 0 &&
+			customOptions.styleType === "toggle" && (
+				<FlexToggleWrapper className={className} $layoutType={customOptions?.layoutType ?? "horizontal"}>
 					{options.map((option, index) => {
 						const radioButtonId = formatId(index);
 
```

Comment:
now that there is a `customOptions.styleType` check, the other checks are not necessary anymore
can apply the same change to `checkbox-group`

```
// old
indicator={customOptions.styleType === "toggle" && customOptions?.indicator}
styleType={
	customOptions.styleType === "toggle" && customOptions?.border === false
		? "no-border"
		: "default"
}

// new
indicator={customOptions?.indicator}
styleType={customOptions?.border === false ? "no-border" : "default"}
```